src/components/QuizListContent.tsx

```diff
@@ -22,40 +22,49 @@ export const QuizListContent = ({ title, options, date, type, onAddOption, onSol
 
   return (
     <QuizList type={type}>
-      <Item>{title}</Item>
+      <QuizText>{title}</QuizText>
       <Item>{options}</Item>
       <Item>{changeDate(date)}</Item>
-      <button onClick={onSolve}>Solve</button>
-      <button onClick={onAddOption}>Add Option</button>
+      <ButtonArea>
+        <SolveButton onClick={onSolve}>
+          Solve <div>&gt;</div>
+        </SolveButton>
+        <AddButton onClick={onAddOption}>
+          Add Option <div>&gt;</div>
+        </AddButton>
+      </ButtonArea>
     </QuizList>
   )
 }
 
 const QuizList = styled.div<{ type?: 'Content' | 'End' }>`
   display: grid;
-  grid-template-columns: auto 110px 130px;
+  align-items: center;
+  grid-template-columns: auto 100px 100px 100px;
+  column-gap: 8px;
   background-color: white;
   place-items: left;
-  padding: 20px;
+  padding: 16px;
   ${typography.b02};
   text-align: center;
 
   ${props =>
     props.type === 'Content' &&
     css`
       border-bottom: 1px solid #dbdbdb;
-      &:hover {
-        background-color: #fafafa;
-        cursor: pointer;
-      }
+      // &:hover {
```

Comment:
are you going to rollback this code at some point later? If not, I think it is better to remove it so that there is no dead legacy code.